service/controller/v9/resource/deployment/template/virtual_network_setup.json

```diff
@@ -87,7 +87,12 @@
               },
               "routeTable":{
                 "id":"[parameters('routeTableID')]"
-              }
+              },
+              "serviceEndpoints": [
```

Comment:
What does it do?

Reply:
https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-service-endpoints-overview